nrf_security/Kconfig

```diff
@@ -0,0 +1,468 @@
+menu "Nordic Security"
+
+config NORDIC_SECURITY_BACKEND
+	bool
+	prompt "Use Nordic provided security backend"
+	depends on SOC_FAMILY_NRF
+	select NRFXLIB_CRYPTO
+	help
+	  Help message
+
+if NORDIC_SECURITY_BACKEND
+
+comment "Backend Selection"
+
+config CC310_BACKEND
```

Comment:
Backend selection should be a `choice` if one and only one should be selected.

Reply:
You can enable both, We will explain this in documentation update (coming later)